### ISSUE TITLE
o jo e nimi "tonsi"

### ISSUE THREAD

**Issue by porglezomp** [jackhumbert/sitelen-pona-pona#6]
mi tonsi. mi wile toki e sitelen pi nimi "tonsi" lon sitelen ale.
sitelen pi "linja pona" li jo e sitelen ni:

<img width="108" alt="image" src="https://user-images.githubusercontent.com/1690225/84193584-abde9300-aa69-11ea-861d-11d9f17838f3.png">

a pona!

-----

Hi! I'm nonbinary and I want to be able to write that in the different toki pona fonts I use. The "linja pona" font writes it like the image above. I would really appreciate if you could add it. Thank you!

**Comment by porglezomp** [comment]
Ah! I see that this was already done in a recent commit! I'll leave this open until there's a release that has it :D

**Comment by porglezomp** [comment]
It looks like you might not have updated the Unicode proposal.

**Comment by jackhumbert** [comment]
Yeah! I'd like to add it to that - I wanted to get some feedback on how things should be organised, ie should all (non-pu and all) of the words be listed there alphabetically.

**Comment by porglezomp** [comment]
I'm no expert, but I would put all the pu words consecutively and alphabetically, then the cartouche end/start and combining, and then all non-pu words after that for neatness in any potential future expansions.

**Comment by porglezomp** [comment]
One question: does sitelen pona need a dedicated combining mark encoding? Might it not just use a normal ZWJ?
